Classes/Domain/Index/Queue/RecordMonitor/Helper/ConfigurationAwareRecordService.php

```diff
@@ -123,37 +123,20 @@ protected function getRecordIfIndexConfigurationIsValid(
     protected function getRecordForIndexConfigurationIsValid(
         string $recordTable,
         int $recordUid,
-        string $recordWhereClause = '',
+        string $recordWhereClause = ''
```

Comment:
```suggestion
        string $recordWhereClause = '',
```

Please put comma back.